src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -768,48 +768,6 @@ public async Task RequestDelegateHandlesNullableStringValuesFromExplicitQueryStr
         Assert.Equal(new StringValues(new[] { "7", "8", "9" }), httpContext.Items["form"]!);
     }
 
-    [Fact]
```

Comment:
I know there has been stuff going on with testing relating to RDG. Does that affect the RDF tests? Is RDF tested with the same code as RDG somewhere else now so we can share tests?